rust/ql/lib/codeql/rust/elements/internal/FormatImpl.qll

```diff
@@ -42,7 +42,9 @@ module Impl {
 
     override int getIndex() { result = index }
 
-    override FormatArgument getArgument() { result.getParent() = this }
+    override FormatArgument getArgument(int i) {
```

Comment:
The old definition of `getArgument()` was clearly wrong. I think I simply forgot to remove the `argument` field from the schema, so we could just delete this predicate. If you want to keep it, that's fine too.

Reply:
If we remove it, then how will we be able to retrieve the arguments?

Reply:
There are some other predicates `getWidthArgument()`, `getPrecisionArgument()`, and `getArgumentRef()` .

Reply:
The `getArgument(index)` predicate is only used in one place, which can easily be rewritten.

Reply:
Alright, I'll remove it then.